starter_project/src/navigation/tag_seek.py

```diff
@@ -0,0 +1,25 @@
+from state import BaseState
```

Comment:
a couple more "hints" could probably be helpful in the nav portion, such as 'refer to context' and things of that nature, but otherwise looks good, very clear directions